yfmos.py

```diff
@@ -223,7 +223,7 @@ def run(self):
             print(e, file=sys.stderr)
             exit(1)
 
-    def __parge_B1(self, b1String):
+    def __parge_B1(self, b1String, device):
```

Comment:
Do not pass `device` as additional parameter. The parsing should only be dependent on the B1 string.